ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/proxy/provision/cucushift-installer-rehearse-aws-ipi-proxy-provision-chain.yaml

```diff
@@ -1,7 +1,13 @@
 chain:
   as: cucushift-installer-rehearse-aws-ipi-proxy-provision
   steps:
-  - chain: ipi-conf-aws-proxy
+  - ref: aws-provision-vpc-disconnected
+  - chain: aws-provision-bastionhost
+  - ref: ipi-conf
+  - ref: ipi-conf-aws-custom-vpc
+  - ref: ipi-conf-proxy
+  - ref: ipi-conf-aws
+  - ref: ipi-install-monitoringpvc
   - chain: ipi-install
```

Comment:
Per yesterday's meeting, suggest to append ref: mirror-images-enable-qe-catalogsource here.